client/components/scroll-to-anchor-on-mount/index.tsx

```diff
@@ -0,0 +1,11 @@
+import { useEffect } from 'react';
+import scrollToAnchor from 'calypso/lib/scroll-to-anchor';
+
+export function ScrollToAnchorOnMount() {
+	useEffect( () => {
+		setTimeout( () => {
+			scrollToAnchor( { offset: 30 } );
```

Comment:
Should we have a note for why `offset=30`? It seems like an arbitrary amount...

Reply:
Good catch, I created a prop.
https://github.com/Automattic/wp-calypso/pull/73982/commits/cf09cc3b8ce01ae682d99c25af18e0e702b74978